scanpy/preprocessing/docs.py

```diff
@@ -0,0 +1,88 @@
+doc_norm_bulk = """\
+Normalize total counts per cell.
+
+Normalize each cell by total counts over genes, so that every cell has
+the same total count after normalization.
+
+Similar functions are used, for example, by Seurat [Satija15]_, Cell Ranger
+[Zheng17]_ or SPRING [Weinreb17]_.
+
+Parameters
+----------
+data : :class:`~anndata.AnnData`
```

Comment:
As all of this is for an `AnnData` and we use `adata` for an `AnnData` throughout, can you please adapt this?